ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -904,3 +906,70 @@ def test_ipahealthcheck_trust_agent_member_check(self):
         for check in data:
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] == self.master.hostname
+
+
+class TestIpaHealthCheckFilesystemSpace(IntegrationTest):
+    """
+    ipa-healthcheck tool test for running low on disk space.
+    """
+    num_replicas = 0
```

Comment:
`num_replicas` can be removed as the definition is present in base class `IntegrationTest`.

Reply:
done